### PR TITLE
GH#892: Service Advertisement BGP, update to recent Metallb configuration

### PR DESCRIPTION
**Pull request by k86td**
Product Version(s):

Calico Open-Source.

Issue:

Github issue #892.

Link to docs preview:

[Deploy-preview-1020](https://deploy-preview-1020--tigera.netlify.app/calico/next/networking/configuring/advertise-service-ips#advertise-service-load-balancer-ip-addresses).

SME review:
- [x] An SME has approved this change. 
Lance approved. 

DOCS review:
- [x] A member of the docs team has approved this change.

Additional information:

Merge checklist:

- [ ] Deploy preview inspected wherever changes were made 
- [x] Build completed successfully
- [x] Test have passed

### CODE REVIEW

**Comment by bmckercher123** [comment]
@k86td Cheers and thank you for submitting this update. We'll get a SME to review, and determine which other versions of the docs require the same update.

File: calico/networking/configuring/advertise-service-ips.mdx [hunk 1]
```diff
@@ -216,22 +216,17 @@ To install the MetalLB controller for allocating addresses, perform the followin
 
    However, do not install the `metallb-system/speaker` component. The speaker component also attempts to establish BGP sessions on the node, and will conflict with Calico.
 
-1. Configure MetalLB to provision addresses by creating the following config map, replacing `x.x.x.x/16` with the CIDR given to {{prodname}} in the steps above.
+1. Configure MetalLB to provision addresses by creating the following IPAddressPool, replacing `x.x.x.x/16` with the CIDR given to {{prodname}} in the steps above. Please note IPAddressPool require Metallb `0.13+`.
```
Comment (bmckercher123) [inline calico/networking/configuring/advertise-service-ips.mdx]:
```suggestion
1. Configure MetalLB to provision addresses by creating the following IPAddressPool, replacing `x.x.x.x/16` with the CIDR given to {{prodname}} in the steps above. Please note IPAddressPool requires Metallb `0.13+`.
```

Comment (bmckercher123) [reply calico/networking/configuring/advertise-service-ips.mdx]: Almost there. After fixing above sentence, please copy the changes to these folders as well:

calico-versioned-docs
   3.26
   3.25

calico-enterprise-docs
   3.19 (root)
calico-enterprise-versioned-docs
   3.18
   3.17

calico-cloud
  3.19 (root)
calico-cloud-versioned-docs
  3.18
  3.17

**Review by bmckercher123 — APPROVED** [review]
Looks good. Bravo!

**Comment by bmckercher123** [comment]
@k86td If you would rebase to squash commits, I'll merge it. Thanks for your patience.

**Comment by k86td** [comment]
Okay, I think I've done it correctly. Keep me posted if there's something else.

Thank you!

**Comment by bmckercher123** [comment]
Perfect! And thanks for your contribution and patience.

**Pull request merged**